packages/tools/devtools/devtools-core/src/Features.ts

```diff
@@ -49,6 +49,11 @@ export enum ContainerDevtoolsFeature {
 	 * the Container.
 	 */
 	ContainerData = "container-data",
+
+	/**
+	 * thing
```

Comment:
The only one I was not able to make internal was enum EditType in commonInterfaces.ts since it is used by editProps, the new field in VisualNodeBase. Any way aorund this or is this fine?

Reply:
Wrong place for the comment

Reply:
Ah, that makes sense. Hmm. We may want to remove `editProps` for the time being. Otherwise, we add options to the public API to enable editing, but that doesn't actually get respected yet by the system. We can comment out that property and leave a TODO to restore it once we're ready to enable editing. We'll have to update some code in the view package with TODOs to accommodate this too.